### DISCUSSION TITLE
Hard to find.

### DISCUSSION THREAD

**Discussion by semi710** [Akylas/OSS-DocumentScanner#53] [General]
it's too hard to find the project as the name `com.akylas.documentscanner`

**Comment by farfromrefug** [comment]
@niksingh710 you mean the project name on github?

**Reply by semi710** [reply]
Yep

**Comment by farfromrefug** [comment]
@niksingh710 it is a good point. Might change it. What do you think it should be ? `DocumentScanner`, `Document-Scanner`, `OSS-Document-Scanner` ?

**Reply by semi710** [reply]
imo `OSS-DocumentScanner` seems pretty good but we can have more input from the community.
Description is on point so it seems fine.

**Reply by farfromrefug** [reply]
Ok i ll wait a few days. If no one propose something better i ll go for this. Thanks @niksingh710 for your great input here!
